src/borg/key.py

```diff
@@ -371,7 +371,7 @@ def encrypt_key_file(self, data, passphrase):
         iterations = PBKDF2_ITERATIONS
         key = passphrase.kdf(salt, iterations, 32)
         hash = hmac_sha256(key, data)
-        cdata = AES(is_encrypt=True, key=key).encrypt(data)
+        cdata = AES(key, b'\0'*16).encrypt(data)
```

Comment:
Now that this is in plain view. Urgh.
Keyblob changes are really not very safe. This is more of a problem for the repokey case (because the server is more untrusted) but it’s bad in every case. Multiple time pad just is bad.